tgui/packages/tgui/interfaces/ArmorValues.tsx

```diff
@@ -0,0 +1,44 @@
+import { useBackend } from '../backend';
+import { Box, Divider, ProgressBar, Section } from '../components';
+import { Window } from '../layouts';
+
+export type ArmorValuesData = {
+  armor_values: string[];
+};
+
+export const ArmorValues = (props, context) => {
+  const { act, data } = useBackend<ArmorValuesData>(context);
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section>
+          <Box>
+            THE STATISTICS BELOW IS OUT OF CHARACTER INFO, YOU CAN USE THIS TO
+            REFERENCE ARMOR VALUES, BUT DO NOT STATE THE PERCENTAGES IN
+            CHARACTER
+          </Box>
```

Comment:
NoticeBox this and make it not caps. Also:
`the statistics below are* out of character info`

Reply:
fixed